src/qt/bitcoin.cpp

```diff
@@ -394,14 +398,36 @@ bool BitcoinApplication::setupPassword(SecureString& password) {
   return false;
 }
 
+// this will be used to get mnemonic words
+bool BitcoinApplication::setupMnemonicWords(std::vector<std::string>& wordlist) {
+    if (gArgs.GetBoolArg("-disablewallet", DEFAULT_DISABLE_WALLET)) {
+        LogPrintf("Wallet disabled!\n");
+    } else {
+        for (const std::string &walletFile : gArgs.GetArgs("-wallet")) {
+            if (fs::exists(walletFile)) return true;
+        }
+    }
+    if (CheckIfWalletDatExists()) return true;
+
+    StartOptionsMain dlg(nullptr);
+    dlg.exec();
+    wordlist = dlg.getWords();
```

Comment:
please change "wordlist" to something else like "words" and also on line 402 to avoid compiler warnings. It shadows a member variable